cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -447,6 +414,11 @@ func TestCompleteOptions(t *testing.T) {
 		},
 	}
 	for _, tc := range testCases {
+		kubernetesOptions, err := getKubernetesOptions(kubeConfigPath)
```

Comment:
What am I even looking at? What is this testing? Why is this here?

Reply:
I do not have a setter on `kubernetesOptions` for `kubeconfig`.
It has to be parsed from args.